nucypher/blockchain/eth/actors.py

```diff
@@ -1306,6 +1306,11 @@ def __init__(self,
         self.rate = rate
         self.duration_periods = duration_periods
 
+    @property
+    def default_rate(self):
+        _minimum, default, _maximum = self.policy_agent.get_min_reward_rate_range()
+        return default
```

Comment:
Originally, Alice's default rate was `self.rate` (L1306), which is used in `BlockchainPolicyAuthor.generate_policy_parameters()` as the fallback rate when no rate is provided. 
We should unify what's the default rate now. If you ask me, I think that having a default rate initialized in Alice's constructor doesn't have much sense now, so I'd prefer to keep this new `default_rate` as the true default rate for Alice. We can add that to the scope of #1709 and move on.

Reply:
Added to #1709